packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -50,6 +50,8 @@ interface CrudMoreActionsGroupProps {
 }
 
 function CrudMoreActionsGroup({ groupTitle, children, menuListProps, typographyProps }: PropsWithChildren<CrudMoreActionsGroupProps>) {
+    if (!groupTitle) return <MenuList {...menuListProps}>{children}</MenuList>;
+
```

Comment:
A task already exists to hide the group-title if there is only one group: [COM-1174](https://vivid-planet.atlassian.net/browse/COM-1174)
So this change should not be necessary. 

```suggestion
```

Reply:
Reverted changes: b73315b2b15cd05017474e0c5166ca29a36a955b